data-generator/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```

Comment:
can we use ubi9 instead?

Reply:
@kornys this was based on the Strimzi base image, if I change it to ubi9 I see the following exception:
```
Step 3/6 : RUN microdnf update     && microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install java-17-openjdk-headless shadow-utils     && microdnf clean all
 ---> Running in 593b7d6dbbf9

(microdnf:8): GLib-ERROR **: 08:58:00.518: file ../glib/gthread-posix.c: line 1338 (g_system_thread_new): error 'Operation not permitted' during 'pthread_create'
The command '/bin/sh -c microdnf update     && microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install java-17-openjdk-headless shadow-utils     && microdnf clean all' returned a non-zero code: 133
```

Do you know what alternative `microdnf` commands we should run for ubi9?

Reply:
upgrading to ubi9 worked for me, maybe an environmental difference is involved? I'm on Fedora 40 using podman with:

```
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
 
 USER root
 
-RUN microdnf update \
-    && microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install java-17-openjdk-headless shadow-utils \
+RUN microdnf -y update \
+    && microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y java-17-openjdk-headless shadow-utils \
```

Kroxylicious has a [ubi9 based image](https://github.com/kroxylicious/kroxylicious/blob/main/Dockerfile) for inspiration.

Reply:
edit: I guess it could depend on your minikube driver too? I'm using the podman driver

Reply:
Looks like it could be my minikube driver, since today I'm having issues running the Strimzi build locally as well

Reply:
I've updated my minikube version and updating to ubi9 is all working fine today, so I've pushed that change. @kornys does the updated Dockerfile look good to you?